### ISSUE TITLE
Ore Dictionary Bug

### ISSUE THREAD

**Issue by spacebuilder2020** [Beyond-Reality/BeyondRealityModPack#409]
Jabba Better Barrels can convert between different types of iron (Wrought vs Regular)

**Comment by leagris** [comment]
Jabba barrel bug
All iron types are registered under fooAnyIron additionally to their specific.
If I'd kill all AnyMaterial registrations (Iron, Copper) it would break a lot of recipes including internals from Gregtech.

**Comment by leagris** [comment]
JABBA Team denied fixing what they attribute to Gregtech misuse of Ore Dictionary.

**Comment by JasonMcRay** [comment]
@leagris I did this, removed the AnyMaterial registrations and so far I haven't found any real issues. There is not many recipes that require ingots. Machine processing recipes are not affected by oredict changes with MT as well.

And even if someone has for example Annealed Copper or Wrought Iron and would like to use it inb a recipe that normally it would work because of the "ingotAny" players still can easily convert it back to normal version.

So what I did I removed the anyMaterial registrations for the "different" ones and left it for the regular. So for example I removed ingotAnyIron from Wrought and Pig Iron but left it on the normal Iron.
